### PR TITLE
feat: 🎸 add pick up call feature

### PR DESCRIPTION
**Pull request by Hazyzh**

### CODE REVIEW

File: src/RingCentralCall.ts [hunk 1]
```diff
@@ -34,6 +47,18 @@ export interface ActiveCallInfo extends ActiveCallInfoBase {
...
+export interface IPickUpCallParams {
+  sessionId: string;
+  toNumber: string;
+  fromNumber: string;
+  serverId: string;
+  telephonySessionId: string;
+  sessionDescriptionHandlerOptions:  { 
+    constraints?: { audio: boolean, video: boolean }
+  }
+  ;
```
Comment (embbnux) [inline src/RingCentralCall.ts]: typo?

File: src/RingCentralCall.ts [hunk 2]
```diff
@@ -119,6 +144,42 @@ export class RingCentralCall extends EventEmitter {
...
+      `RC-call-type: inbound-pickup; session-id: ${sessionId}; server-id: ${serverId}`,
+    ];
+    const inviteOptions = {
+      sessionDescriptionHandlerOptions,
+      fromNumber,
+      extraHeaders,
+    };
+    this._webphoneInviteFromSDK = true;
+    const webphoneSession =
+    this._webphone &&
```
Comment (embbnux) [inline src/RingCentralCall.ts]: space here

Comment (Hazyzh) [reply src/RingCentralCall.ts]: added

File: test/ringcentral-call.test.ts [hunk 3]
```diff
@@ -511,4 +582,8 @@ describe('RingCentral Call ::', () => {
       expect(inviteSent).toBe(true);
     });
   });
+
+  describe('pickupInboundCall scenarios tests', () => {
```
Comment (embbnux) [inline test/ringcentral-call.test.ts]: remove unused "describe"

**Pull request merged**